Content: update docs headless home and playwright / puppeteer basics (#668)
checkly/checklyhq.com: site/content/learn/headless/basics-playwright-intro.md

The [official documentation](https://playwright.dev/) for Playwright reads:

> > Playwright provides a set of APIs to automate Chromium, Firefox and WebKit browsers. By using the Playwright API, you can write JavaScript code to create new browser pages, navigate to URLs and then interact with elements on a page. 

Reviewer: Let's mention TypeScript, too. :) 

+ it might be worth sneaking in a note that .net, python etc also work whereas we're only focussing on the JS/TS side of things.

> You can import [test](https://playwright.dev/docs/api/class-test) from Playwright which provides a test function to declare tests and an expect function to write assertions. We'll be using this throughout our e2e examples as a best practice. 

Reviewer: ```suggestion
You can import Playwright's provided [`test`](https://playwright.dev/docs/api/class-test) and `expect` functions to declare tests and add assertions. We'll be using these throughout our e2e examples as a best practice. 
```

☝️ add links to `expect`.

Just tidying thing a bit. ;)
